index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width">
+        <meta name="author" content="Amos Gloria">
+        <title>Facebook Clone | Welcome</title>
+        <link rel="stylesheet" href="./css/style.css">
+        </head>
+        <body>
+            <header>
+                <div class="header-block">
+                    <div id="branding">
+                        <h1><span class="highlight">facebook</span></h1>
+                        </div>
+                           <nav>
+                                <div class="email">
+                                <label>Email or Phone</label><br>
+                                <input type="email or phone" placeholder="Email"><br>
+                                <label><input type="checkbox" id="mycheck" onclick="myFunction()">Keep me logged in</label>
+                                </div>
+                                <div class="password">
+                                   <label>Password</label><br>
+                                   <input type="password" placeholder="Password"><br>
+                                   <a href="#">Forgotten your password?</a> 
+                                </div>
+                                <div class="login-button">
+                                    <input type="submit" value="Login">
+                                </div>
+                            </nav>
+                            </div>
+                            </header>
+                            <section>
+                                <div class="content-control">
+                                    <div class="left">
+                                    <h4>Facebook helps you connect and share with the people in your life.</h4>
+                                    <img src="img/world.jpg"/>
+                                    </div>
+                                    <div class="right">
+                                        <h1>Create an account</h1>
+                                        <p>It's free and always will be.</p>
```

Comment:
Reduce the size of this a little. Also make the font style same as the one used for _Forgotten your Password_ (Times New Roman i guess). Compare with the prototype given